### PR TITLE
Updated Metrics Storage Dashboard

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-10059 DOC-10545 DOC-10589

In ui-storage-dashboard.md, (1) removed links to outdated images, (2) corrected metric name in Capacity graph (3) added sections for new graphs L0 SSTable Count, L0 SSTable Size, Disk Write Breakdown. (4) updated Other graphs section, (5) added section for Per store metrics. Deleted outdated images.

Rendered preview

- [Storage Dashboard](https://deploy-preview-19093--cockroachdb-docs.netlify.app/docs/v24.3/ui-storage-dashboard.html)

### CODE REVIEW

**Comment by florence-crl** [comment]
Hi @jbowens , I assigned you to review this docs PR since you are storage team tech lead. Please re-assign if needed. thanks

File: src/current/v24.3/ui-storage-dashboard.md [hunk 1]
```diff
@@ -77,24 +83,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
...
 
+## Disk Write Breakdown
+
+- In the node view, the graph shows the number of bytes written to disk per second categorized according to the source for that node.
+
+- In the cluster view, the graph shows the number of bytes written to disk per second categorized according to the source for each node.
+
+Possible sources of writes with their series label are:
+
+- [WAL]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) (`pebble-wal`)
```
Comment (jbowens) [inline src/current/v24.3/ui-storage-dashboard.md:94]: The link here should probably be to the `#memtable-and-write-ahead-log` page heading

File: src/current/v24.3/ui-storage-dashboard.md [hunk 2]
```diff
@@ -77,24 +83,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
...
+- [Compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) (`pebble-compaction`)
```
Comment (jbowens) [inline src/current/v24.3/ui-storage-dashboard.md:95]: whereas this one should be to `#compaction`

**Review by jbowens — APPROVED** [review]
lgtm; just a couple nits around links pointing to the appropriate heading within the storage layer architecture page

File: src/current/v24.3/ui-storage-dashboard.md (hunk 1 above)
Comment (florence-crl) [reply src/current/v24.3/ui-storage-dashboard.md]: good catch, corrected link

File: src/current/v24.3/ui-storage-dashboard.md (hunk 2 above)
Comment (florence-crl) [reply src/current/v24.3/ui-storage-dashboard.md]: good catch, corrected link

**Review by florence-crl — COMMENTED** [review]
TFTR

File: src/current/v24.3/ui-storage-dashboard.md [hunk 3]
```diff
@@ -13,17 +13,21 @@ To view this dashboard, [access the DB Console]({% link {{ page.version.version
 
 {% include {{ page.version.version }}/ui/ui-metrics-navigation.md %}
 
+### Per store metrics
+
+To display per [store]({% link {{ page.version.version }}/cockroach-start.md %}#store) metrics, select a specific node in the **Graph** menu. An aggregate metric for that node as well as a metric for each store of that node will be displayed for [L0 SSTable Count](#l0-sstable-count), [L0 SSTable Size](#l0-sstable-size), and some [other graphs](#other-graphs).
+
+---
```
Comment (kathancox) [inline src/current/v24.3/ui-storage-dashboard.md:20]: Is the break line intentional? I've never seen this included on a docs page before.

File: src/current/v24.3/ui-storage-dashboard.md [hunk 4]
```diff
@@ -77,24 +89,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
...
+- [SSTable ingestions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#ssts) (`pebble-ingestion`)
+- [Memtable flushes]({% link {{ page.version.version }}/architecture/storage-layer.md %}#memtable-and-write-ahead-log) (`pebble-memtable-flush`)
+- [Raft snapshots]({% link {{ page.version.version }}/architecture/replication-layer.md %}#snapshots) (`raft-snapshot`)
+- [Encryption Registry]({% link {{ page.version.version }}/security-reference/encryption.md %}#encryption-keys-used-by-cockroachdb-self-hosted-clusters) (`encryption-registry`)
+- [Logs]({% link {{ page.version.version }}/logging-overview.md %}) (`crdb-log`)
+- SQL row spill (`sql-row-spill`), refer to [`cockroach start` command]({% link {{ page.version.version }}/cockroach-start.md %}#flags) flag `--max-disk-temp-storage`
+- [SQL columnar spill]({% link {{ page.version.version }}/vectorized-execution.md %}#disk-spilling-operations) (`sql-col-spill`)
+
+Refer to the **Hardware** dashboard [**Disk Write Bytes** graph]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#disk-write-bytes) to view an aggregate of all disk writes.
```
Comment (kathancox) [inline src/current/v24.3/ui-storage-dashboard.md:110]:
```suggestion
To view an aggregate of all disk writes, refer to the **Hardware** dashboard [**Disk Write Bytes** graph]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#disk-write-bytes).
```

Comment (kathancox) [reply src/current/v24.3/ui-storage-dashboard.md:110]: I think pre-empting the link with the reason to follow the link is helpful for the reader. This is non-blocking, so take the suggestion if you want!

File: src/current/v24.3/ui-storage-dashboard.md [hunk 5]
```diff
@@ -77,24 +89,46 @@ If you are running multiple nodes on a single machine (not recommended), the act
...
+WAL Bytes Written | Bytes written to WAL files across all nodes.
 Compactions | Bytes written by [compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) across all nodes.
-Ingestions | Bytes written by SSTable injections across all nodes.
+Ingestions | Bytes written by SSTable ingestions across all nodes.
 Write Stalls | The number of intentional write stalls per second across all nodes, used to backpressure incoming writes during periods of heavy write traffic.
 Time Series Writes | The number of successfully written time-series samples, and number of errors attempting to write time series samples, per second across all nodes.
-Time Series Bytes Written | The number of bytes written by the time-series system per second across all nodes.
+Time Series Bytes Written | The number of bytes written by the time-series system per second across all nodes.<br><br>Note that this does not reflect the rate at which disk space is consumed by time series; the data is highly compressed on disk. This rate is instead intended to indicate the amount of network traffic and disk activity generated by time series writes.
```
Comment (kathancox) [inline src/current/v24.3/ui-storage-dashboard.md:131]: when "time series" is modifying a noun, should it be "time-series"? There is "time-series system" and "time series writes" here — not sure which one you want to go for?

**Review by kathancox — APPROVED** [review]
LGTM, couple of non-blocking nits!

**Review by florence-crl — COMMENTED** [review]
tftr

**Pull request merged**